src/components/Classes/index.js

```diff
@@ -1,41 +1,39 @@
-import React from "react";
+import React from 'react';
+import { ThumbnailArray } from "../../constants";
 import * as fetch from "../../lib/fetch.js";
 import ClassBox from "./components/ClassBox";
 import ConfirmLeaveModalContainer from "./containers/ConfirmLeaveModalContainer";
 import CreateClassModalContainer from "./containers/CreateClassModalContainer";
 import JoinClassModalContainer from "./containers/JoinClassModalContainer";
 import OpenPanelButtonContainer from "../common/containers/OpenPanelButtonContainer";
-import { ThumbnailArray } from "../../constants";
-import "../../styles/ClassBox.scss";
-import "../../styles/Classes.scss";
-import "../../styles/Login.scss";
-import LoadingPage from "../common/LoadingPage";
+import '../../styles/ClassBox.scss';
+import '../../styles/Classes.scss';
+import '../../styles/Login.scss';
+import LoadingPage from '../common/LoadingPage';
 import { getInstructorString } from '../../util/classes';
 
-import { Button } from "reactstrap";
+import { Button } from 'reactstrap';
 
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faPlus } from "@fortawesome/free-solid-svg-icons";
-import { faKey } from "@fortawesome/free-solid-svg-icons";
-import { faPencilAlt } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus , faKey , faPencilAlt } from "@fortawesome/free-solid-svg-icons";
 
 class Classes extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       loaded: false,
-      redirectTo: "",
+      redirectTo: '',
       confirmLeaveModalOpen: false,
       createClassModalOpen: false,
       joinClassModalOpen: false,
-      selectedClass: "",
-      selectedImg: "",
-      selectedCid: "",
-      classCode: "",
+      selectedClass: '',
+      selectedImg: '',
+      selectedCid: '',
+      classCode: '',
     };
   }
 
-  componentDidMount = async () => {
+  componentDidMount() {
```

Comment:
Let's just revert this line
```suggestion
  componentDidMount = async () => {
```